### PR TITLE
Replace jenkins config with drone

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?

Replaces Jenkins config with drone, and adds kubernetes file to describe cronjob.

#### What problem is this solving?

This will enable us to have a finer control over our tests cronjob.

#### How should this be manually tested?

See CI output.

#### Screenshots or example usage

N/A

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+kind: pipeline
```
Comment (wdsrocha) [inline .drone.yml:1]: wow, good job! Did you use any other `.drone.yml` as a base for this? Which one?

Comment (lucasecdb) [reply .drone.yml:1]: yep! I used the one I created for [fastcheckout](https://github.com/vtex/fastcheckout/blob/main/.drone.yml)

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  pull: if-not-exists
+  environment:
+    DOCKER_REGISTRY: 558830342743.dkr.ecr.us-east-1.amazonaws.com
+  commands:
+    - docker build --network=host -t $$DOCKER_REGISTRY/healthcheck/webtests/checkout:latest -f ./dockerfiles/stable/Dockerfile .
+  volumes:
+    - name: dockersock
+      path: /var/run
+- name: publish
+  image: 558830342743.dkr.ecr.us-east-1.amazonaws.com/reliability/jenkins/docker:latest
```
Comment (wdsrocha) [inline .drone.yml:17]: hmm how does this works? why do we need jenkins?

File: kubernetes/stable.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    secretsManager:
+      enabled: true
+      secretsName: stable/checkout-ui-tests/checkout-ui-tests
+    version: latest
+  apps:
+  - name: checkout-tests-cronjob
+    kind: cronjob
+    schedule: "* * * * *"
+    activeDeadlineSeconds: 2400
+    startingDeadlineSeconds: 6000
```
Comment (wdsrocha) [inline kubernetes/stable.yml:16]: Could you explain the rationale behind those numbers?

**Comment by wdsrocha** [comment]
Suggestion: How about adding the build status badge on README? I think this makes it easier for new devs to understand what/where the build is happening.

The badge: [Build Status](https://drone-robots.vtex.com/vtex/checkout-ui-tests)

File: .drone.yml (hunk 2 above)
Comment (lucasecdb) [reply .drone.yml:17]: it doesn't include jenkins, it is the image the SRE team has created that includes docker and a script to authenticate to AWS (you can see it in use in the `commands` section)

File: kubernetes/stable.yml (hunk 3 above)
Comment (lucasecdb) [reply kubernetes/stable.yml:16]: I got them from [here](https://github.com/vtex/io-sre-konfig/blob/master/services/100-cronjobs/004-healthcheck/013-healthcheck-checkout-ui.yml#L9-L12), which is the config for the tests that are running currently

File: .drone.yml (hunk 2 above)
Comment (wdsrocha) [reply .drone.yml:17]: ah I see, so this image is just placed weirdly at the `jenkins` directory, right?

**Review by wdsrocha — APPROVED** [review]
really nice addition ✨

File: .drone.yml (hunk 2 above)
Comment (lucasecdb) [reply .drone.yml:17]: yep

**Review by jeffersontuc — APPROVED** [review]

**Review by lbebber — APPROVED** [review]

**Pull request merged**